nncf/experimental/common/quantization/algorithms/range_estimator/range_estimator.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2024 Intel Corporation
```

Comment:
I would suggest renaming `range_estimator.py` -> `algorithm.py` to align with other algorithms' implementations.